code/modules/clothing/spacesuits/rig.dm

```diff
@@ -238,6 +238,7 @@
 	wizard_garb = 1
 
 	species_restricted = null
+	allowed = list(/obj/item/weapon/teleportation_scroll,/obj/item/weapon/gun/energy/staff)
```

Comment:
YOU WANT OXYGEN TANK?  FUCK YOU HAVE STAFF INSTEAD

Reply:
This file is non oop garbage with allowed lists.

Reply:
real comment hours:  You're overriding rigs default allowed list which is inherited in other objects like rig/mining with a new allowed list but not including the things you're overriding, leading to loss of behavior.  look at the syndie suit for a usage example
``` allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/weapon/gun,/obj/item/ammo_storage,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/melee/energy/sword,/obj/item/weapon/handcuffs) ```